packages/gatsby/src/utils/webpack-utils.js

```diff
@@ -194,14 +194,14 @@ module.exports = async ({
 
     css: (options = {}) => {
       return {
-        loader: isSSR
-          ? require.resolve(`css-loader/locals`)
-          : require.resolve(`css-loader`),
+        loader: require.resolve(`css-loader`),
         options: {
           sourceMap: !PRODUCTION,
-          camelCase: `dashesOnly`,
-          // https://github.com/webpack-contrib/css-loader/issues/406
-          localIdentName: `[name]--[local]--[hash:base64:5]`,
+          localsConvention: `dashesOnly`,
+          modules: {
+            mode: `local`,
+            localIdentName: `[name]--[local]--[hash:base64:5]`,
+          },
```

Comment:
These changes are all accurate for css-loader@3. E.g. there is no `/locals` file anymore, `camelCase` was changed to `localsConvention`, `modules` is setup correctly.

One thing I'm seeing here, that's old behavior but something that we could improve is to change the localIdentName to just a hash when we're building for production (unless that's already done via some other mechanism). I'm not arguing for that to be in this PR, just stating so we can track it.